virt_up/instance.py

```diff
@@ -518,7 +518,9 @@ def wait_for_port(self, port):
                 s.settimeout(2)
                 s.connect((address, int(port)))
                 return True
-            except:
+            except Exception as e:
+                if str(e) == "[Errno 111] Connection refused":
+                    time.sleep(2) 
                 pass
```

Comment:
This pass is no longed needed, since you added a statement. (pass is just a no-op placeholder)